modules/openapi-generator/src/main/resources/python/model.mustache

```diff
@@ -215,10 +229,13 @@ class {{classname}}(object):
         if not isinstance(other, {{classname}}):
             return False
 
-        return self.__dict__ == other.__dict__
+        return self.to_dict() == other.to_dict()
 
     def __ne__(self, other):
         """Returns true if both objects are not equal"""
-        return not self == other
```

Comment:
Performance would `not self == other` be better?

Reply:
This again relates to the configuration object. Since Test("abc") will not be equal to Test("abc") because config is initialized separately for both the objects.

Reply:
@slash-arun thanks for the explanation. I wonder if you can add test cases to https://github.com/OpenAPITools/openapi-generator/tree/master/samples/client/petstore/python/tests to cover the issue moving forward.

Reply:
Sure, will work on the test cases